spring-boot-example/mvnw

```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
```

Comment:
Why have these scripts changed?

Reply:
Executing `./mvnw` did not work, as the required file in `.mvn` was missing. I regenerated the wrapper script with `mvn` version `3.8.4` using `mvn wrapper:wrapper`, which also changed the existing files.